src/style/foundation/video-calling.less

```diff
@@ -142,7 +142,7 @@
     display: grid;
     padding: 0.4rem;
     border-radius: 1rem;
-    background-color: #fff;
+    background-color: var(--app-bg-secondary);
```

Comment:
` var(--app-bg-secondary)` seems to fit here, white in light mode, black in dark mode